Model/Api.php

```diff
@@ -1814,6 +1814,30 @@ public function kickoffParallelCampaigns(Contact $contact, &$campaignIds = [], $
             return $this;
         }
 
+        $ramRequiredPercent = intval($this->getIntegrationSetting('parallel_ram', 20));
+        if ($ramRequiredPercent > 0) {
+            try {
+                $memInfo = file_get_contents('/proc/meminfo');
```

Comment:
It'd be a good idea to make this compatible with Windows.

Reply:
I didn't worry about windows compatibility, because of https://github.com/TheDMSGroup/mautic-contact-source/pull/153/files/976c7a33030d3519ceee85122ca65afb68d64b5e#diff-dfdacbe241de7be4998d058612528828R1811

Multithreading is not a thing in windows php, so no point in checking ram there.